scripts/tests/list_tests

```diff
@@ -1,26 +1,59 @@
 #!/usr/bin/env python
-
+# This script will print the list of test classes in
```

Comment:
I haven't figured out how to make discover work in python3,  as a consequence the list of tests is hardcoded here - the list is confirmed when testing is done with python2 so any problems should be caught fairly quickly.  But the A_RunUnitTests also uses discover and are not working in python3.   The problem with discover in list_tests is only encountered with cmake/ctest but the A_RunUnitTests problem happens even when scripts_regression_tests.py is run directly (tests are not run but report as passing).